riscv/csrs.cc

```diff
@@ -419,7 +419,7 @@ reg_t base_status_csr_t::compute_sstatus_write_mask() const noexcept {
     | (proc->extension_enabled('S') ? (SSTATUS_SIE | SSTATUS_SPIE | SSTATUS_SPP) : 0)
     | (has_page ? (SSTATUS_SUM | SSTATUS_MXR) : 0)
     | (has_fs ? SSTATUS_FS : 0)
-    | (proc->any_custom_extensions() ? SSTATUS_XS : 0)
+    | (proc->extension_enabled('X') ? SSTATUS_XS : 0)
```

Comment:
The commit message for b26a2063384786752f7643a403369cbaf01539d4 should explain _why_ you are making this change, and what effect it will have.

Reply:
replace any_custom_extensions() to extension_enabled('X'), so that we can use --isa="xdummy" to enable related function, instead of loading a real extension.

Reply:
I don't understand what that means, but if you think that's a sufficient explanation, please update the commit message for b26a2063384786752f7643a403369cbaf01539d4 with that explanation.